util/fhir.ts

```diff
@@ -37,6 +39,14 @@ export function createPatientResourceString(birthDate: string): string {
   return JSON.stringify(pt, null, 2);
 }
 
+export function getFhirResourceString(resource: fhir4.Resource) {
+  const primaryCodePath = parsedPrimaryCodePaths[resource.resourceType].primaryCodePath;
+  return `(${fhirpath.evaluate(resource, `${primaryCodePath}.coding.code`)}: ${fhirpath.evaluate(
```

Comment:
If the code at the path is not populated, this just returns `(: )`. Suggest returning perhaps the `id` of the resource or nothing.